flows/MaxDecision_Flow.flow

```diff
@@ -0,0 +1,165 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
### pelle WS Coding Standards: Excessively complex flow
#### Finding
Flow \`MaxDecision\\\_Flow\` has an excessive number of decisions. Consider using code for better testability and maintainability.